README.md

```diff
@@ -11,3 +11,14 @@ There are currently two supported versions:
 
 * 1.9.x  - SOLR 4 and Cassandra 1.2.x. See the 1.9.x branch.
 * 2.x - SOLR 7 with SOLR Cloud support and Cassandra 3.x. See the master branch.
+
+## Integration Tests
+
+Integration testing is supported using docker containers.
+To start the required containers, run the following:
+
+```
+docker-compose -f src/test/docker/solr7-cassandra3.yml up -d
+```
```

Comment:
This also needs the instruction from .travis.yml for killing the docker-compose after finished working with it:

```
docker-compose -f src/test/docker/solr7-cassandra3.yml kill
```

Reply:
good idea. Added